### PR TITLE
feat(gradle): bump version 7.4

### PR DESCRIPTION
**Pull request by stchar**
Update build scripts to support gradle >=7.0
Update gradle wrapper to 7.4 

- [ x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ x] Ensure that the pull request title represents the desired changelog entry
- [ x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: buildSrc/src/main/groovy/com/lesfurets/gradle/CopyDependenciesToMavenTreeTask.groovy [hunk 1]
```diff
@@ -30,5 +30,6 @@ class CopyDependenciesToMavenTreeTask extends DefaultTask {
                 from artifact.file
             }
         }
+        //return getOutputDir()
```
Comment (nre-ableton) [inline buildSrc/src/main/groovy/com/lesfurets/gradle/CopyDependenciesToMavenTreeTask.groovy:33]: What's this all about?

**Review by nre-ableton — COMMENTED** [review]
Just one question

File: buildSrc/src/main/groovy/com/lesfurets/gradle/CopyDependenciesToMavenTreeTask.groovy (hunk 1 above)
Comment (stchar) [reply buildSrc/src/main/groovy/com/lesfurets/gradle/CopyDependenciesToMavenTreeTask.groovy:33]: @nre-ableton  `CopyDependenciesToMavenTreeTask` copies `@Grab` dependencies to m2 cache,  we had some tests with `@Grab` usage 

But I can not find any test with `@Grapes or `@Grab` usage. 
I believe as we have no tests we can remove gradle build scripts and grape related configuration as well.

Comment (nre-ableton) [reply buildSrc/src/main/groovy/com/lesfurets/gradle/CopyDependenciesToMavenTreeTask.groovy]: Great, thanks for the cleanup.

**Review by nre-ableton — APPROVED** [review]

**Pull request merged**